stdlib/private/StdlibUnittestFoundationExtras/StdlibUnittestFoundationExtras.swift

```diff
@@ -72,7 +72,7 @@ public func withOverriddenLocaleCurrentLocale<Result>(
 public func autoreleasepoolIfUnoptimizedReturnAutoreleased(
   invoking body: () -> Void
 ) {
-#if arch(i386) && (os(iOS) || os(watchOS))
+#if arch(i386) && targetEnvironment(simulator)
```

Comment:
I'm not sure how this is equivalent to the previous OS conditional.

Reply:
Gah, it isn't, I have that the wrong way around. Thanks, will fix.